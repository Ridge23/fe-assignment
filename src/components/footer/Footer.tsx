import React from 'react';

import './sass/footer.scss';

export default function Footer(): JSX.Element {
    return (
        <footer className="footer">
            <hr className="footer__divider" />
            Â© IT Promocodes, 2019
        </footer>
    )
}