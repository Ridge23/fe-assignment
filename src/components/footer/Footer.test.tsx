import React from 'react';
import { shallow, configure } from 'enzyme';

import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

import Footer from './Footer';

describe('Footer', () => {
  it('should render correctly', () => {
    const component = shallow(<Footer />);
    const currentYear = new Date().getFullYear();

    expect(component).toMatchSnapshot();
    expect(component.text()).toContain('IT Promocodes, ' + currentYear);
  });
});