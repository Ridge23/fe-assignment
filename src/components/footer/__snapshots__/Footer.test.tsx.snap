// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<footer
  className="footer"
>
  <hr
    className="footer__divider"
  />
  Â© IT Promocodes, 
  2021
</footer>
`;
